tests/manager_tests/conftest.py

```diff
@@ -3,6 +3,7 @@
 import os
 import shutil
 import tempfile
+from unittest.mock import patch
```

Comment:
please use the `Monkeypatch` fixture instead of unittest

Reply:
done